src/App.tsx

```diff
@@ -1,27 +1,54 @@
-import React from 'react';
+/* eslint-disable no-console */
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+  const getAllGoods = () => {
+    getAll().then(goodsList => setGoods(goodsList));
+  };
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  const getFirstGoods = () => {
+    get5First().then(goodsList => setGoods(goodsList));
+  };
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+  const getRedColor = () => {
+    getRedGoods().then(goodsList => setGoods(goodsList));
+  };
```

Comment:
instead three functions with the same logic suggest to make one fuction factory 
```
const getGoods = (callback) => () => callback().then(goodsList => setGoods(goodsList));
```
or 
```
 const getGoods = (callback) => async () => setGoods(await callback());
```